project/plugins.sbt

```diff
@@ -20,3 +20,5 @@ addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.5.1")
 addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.4")
```

Comment:
do we need to get approval for plugins? If so I guess we can always remove this plugin now that we have added all the type annotations

Reply:
cc @vivekhmrc

Reply:
@brendanstraughan / @robynHM I checked and it is being used in a lot other repositories so I think we are good :-)